src/resource/page/ResourcePage.test.tsx

```diff
@@ -27,6 +28,21 @@ const testResource: Resource = {
   },
 };
 
+const testDatePeriod: DatePeriod = {
+  id: '1',
+  created: '20-16-2020',
+  modified: '20-16-2020',
+  is_removed: false,
+  name: { fi: '', sv: '', en: '' },
+  description: { fi: '', sv: '', en: '' },
+  start_date: '',
+  end_date: '20-11-2020',
```

Comment:
Not sure if this date format was supported? I know that atleast this one is: yyyy-MM-dd. Though this is just test data so not as important for it to be accurate. Maybe if its applied as documentation how things work it might be of some value to use correct format. Though maybe api supports multiple formats?

Reply:
I'll change these